advanced_functionality/autogluon-multimodal-containers/AutoGluon_Multimodal_SageMaker_Containers.ipynb

```diff
@@ -0,0 +1,595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "519a2c14-1ecf-48f6-b6af-debb4a599345",
+   "metadata": {},
+   "source": [
+    "# AutoGluon Multimodal with Deep Learning Containers on SageMaker\n",
+    "\n",
+    "[AutoGluon](https://github.com/awslabs/autogluon) automates machine learning tasks enabling you to easily achieve strong predictive performance in your applications. With just a few lines of code, you can train and deploy high-accuracy deep learning models on tabular, image, and text data.\n",
+    "This example shows how to use AutoGluon-Multimodal with Amazon SageMaker by applying [pre-built deep learning containers](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#autogluon-training-containers)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e373e83-6681-4337-8e4c-69768102ddc0",
+   "metadata": {},
+   "source": [
+    "# Prerequisites"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62626ac2-e29e-4e1f-9cfd-cd448cda56fb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!python -m pip install --upgrade pip\n",
+    "!python -m pip install -U 'sagemaker>=2.80'"
```

Comment:
This is old, must be: `>=2.103.0`

Reply:
I updated this in the new version. @gradientsky, how did you know it needed to be `2.103.0` as opposed to another version? I see `2.113` is available.